frontend/src/Sidebar.res

```diff
@@ -11,30 +11,51 @@ let pullToString = ((pull_number, branch)) =>
   | None => "#" ++ Belt.Int.toString(pull_number)
   }
 
-let pullsMenu = (~pulls) => {
-  <Column>
-    <Text color=Sx.gray700 weight=#bold uppercase=true size=#md> {Rx.text("Pull Requests")} </Text>
-    {pulls
-    ->Belt.Array.mapWithIndex((i, pull) =>
-      <Link key={string_of_int(i)} href={linkForPull(pull)} text={pullToString(pull)} />
-    )
-    ->Rx.array}
-  </Column>
+module PullsMenu = {
+  @react.component
+  let make = (~pulls, ~repo, ~selectedPull=?) => {
+    <Column>
+      <Text color=Sx.gray700 weight=#bold uppercase=true size=#md>
+        {Rx.text("Pull Requests")}
+      </Text>
+      {pulls
+      ->Belt.Array.mapWithIndex((i, pull) => {
+        let (pull_number, _) = pull
+        <Link
+          active={selectedPull->Belt.Option.mapWithDefault(false, selectedPullNumber =>
+            selectedPullNumber == pull_number
+          )}
+          key={string_of_int(i)}
+          href={linkForPull(repo, pull)}
+          text={pullToString(pull)}
+        />
+      })
+      ->Rx.array}
+    </Column>
+  }
 }
 
 @react.component
-let make = (~url: ReasonReact.Router.url, ~pulls, ~onSynchronizeToggle, ~synchronize) => {
-  let repo_name = "mirage/index"
+let make = (
+  ~pulls,
+  ~selectedRepo,
+  ~repos,
+  ~onSelectRepo,
+  ~onSynchronizeToggle,
+  ~synchronize,
+  ~selectedPull=?,
+) => {
   <Column spacing=Sx.xl2 sx=Styles.sidebarSx>
-    <a className={Sx.make([Sx.text.noUnderline, Sx.text.color(Sx.black)])} href="/#">
-      <Row spacing=Sx.lg>
-        <Icon sx=[Sx.unsafe("width", "48px"), Sx.mt.md] svg=Icon.ocaml />
-        <Column spacing=#px(-6)>
-          <Heading level=#h4 sx=[Sx.m.zero] text=repo_name />
-          <Text size=#sm color=Sx.gray800> {Rx.text("Benchmarks")} </Text>
-        </Column>
-      </Row>
-    </a>
-    {pullsMenu(~pulls)}
+    <select
```

Comment:
Can we use the `Components.Select` here? I'm not very happy with all the components, but have plans to improve them at some point. For now would be good to keep the UI somewhat consistent.

Reply:
I should have checked if something was already existing for that. Thanks!